lib/p2p/Peer.ts

```diff
@@ -511,7 +538,16 @@ class Peer extends EventEmitter {
       this.close();
     });
 
-    this.socket!.on('data', this.parser.feed);
+    this.socket!.on('data', (data) => {
+      this.lastRecv = Date.now();
+      const dataStr = data.toString();
+      if (this.nodePubKey !== undefined) {
+        this.logger.trace(`Received data from ${this.nodePubKey}: ${dataStr}`);
+      } else {
```

Comment:
I'm not sure if there's value in logging the raw traffic, even on a trace level. I figure it will be incomprehensible and that it will quickly flood logs/consoles when there's network activity.

Reply:
Sorry, I added it for myself and forgot removing.